porting/first-steps.rst

```diff
@@ -57,11 +57,12 @@ Arch
 ^^^^^^^^^^
 
 If you have a pure ``amd64`` running you need to add ``[multilib]`` repository to your ``/etc/pacman.conf`` . This will allow you to install and run ``i686`` packages. Please refer to `<https://wiki.archlinux.org/index.php/multilib>`_
+Assuming you have already installed ``base-devel`` meta package; if not please install ``base-devel``.
```

Comment:
`base-devel` is not a meta package, it's a package group. a person can't use AUR without it installing, i don't think we have to point this out at all